src/main/webapp/css/welcome.css

```diff
@@ -0,0 +1,301 @@
+:root {
+    --text-01: #000000;
+    --light-01: #F9F9F9;
+    --light-02: #FFFFFF;
+    --brand-01: #F9F9F9;
+    --brand-02: #f7cd40;
+    --hover-timing: all 0.2s ease;
+    -webkit-font-smoothing: antialiased;
+    --pg-margin: 80px;
+    font-smoothing: antialiased;
+    scroll-behavior: smooth;
+}
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+svg {
+    width: 1.2em;
+    height: 1.2em;
+    stroke: var(--text-01);
+}
+
+svg.bad {
+    stroke: none;
+    color: var(--text-01);
+}
+
+svg #icon-profile.bad {
+    width: 0.5em;
+    height: 0.5em;
+}
+
+a {
+    text-decoration: none;
+    color: var(--text-01);
+}
+
+body {
+    font-family: 'Helvetica Neue', sans-serif;
+    font-weight: 500;
+    transition: var(--hover-timing);
+    background: var(--light-01);
+    width: 100vw;
+    overflow-x: hidden;
+}
+
+header {
+    height: auto;
+    position: sticky;
+    top: 0;
+    border: 1px solid var(--brand-01);
+    z-index: 100;
+}
+
+nav {
+    color: var(--text-01);
+    font-weight: 600;
+    height: 64px;
+    background: var(--light-01);
+    display: flex;
+    flex-direction: row;
+    /*align-items: center;*/
+    /*justify-content: space-between;*/
+    padding: 0 var(--pg-margin);
+    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+    z-index: 100;
+}
+
+nav ul.navigation-menu {
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+    justify-content: center;
+    position: relative;
+    top: 0;
+    z-index: 1;
+    background: none;
+    width: auto;
+    position: relative;
```

Comment:
```suggestion
```
You've duplicated the position style here. You might want to delete the second one.